### DISCUSSION TITLE
I'd like to change the superadmin name and password

### DISCUSSION THREAD

**Discussion by yenson-lau** [Pryaxis/TShock#1726] [General]
Hey guys! It's my first time running a tshock server, and as a result I made the superadmin account too similar to my login account. I'd like to change it to a user like `super` in case I need to transfer the account to someone else. Is there anything I can do?

**Comment by QuiCM** [comment]
I think the easiest way will be to remove your current superadmin account and add a new one with the desired name :)
New users can be added with
`user add username password group`
and deleted with
`user del username`

**Comment by yenson-lau** [comment]
Ah that's super easy, thanks! I thought superadmin wasn't a group for some reason.
